app/views/alchemy/admin/pictures/_filter_and_size_bar.html.erb

```diff
@@ -7,8 +7,10 @@
       in_dialog: true,
       redirect_url: alchemy.admin_pictures_path(
         size: search_filter_params[:size],
-        filter: 'last_upload',
-        form_field_id: @form_field_id
+        filter: { filter: 'last_upload' },
```

Comment:
Shouldn't this be?

```suggestion
        filter: { misc: 'last_upload' },
```

Reply:
oh yes, I missed that. Good catch! Thank you!

I'll incorporate this change into the commit it belongs to.